web/components/answers/answers-graph.tsx

```diff
@@ -18,19 +18,20 @@ export const AnswersGraph = memo(function AnswersGraph(props: {
 }) {
   const { contract, bets, height } = props
   const { createdTime, resolutionTime, closeTime, answers } = contract
+  const now = Date.now()
```

Comment:
wouldn't Date.now() still get recalculated every time the graph gets rerendered?

I would have done `const now = useMemo(Date.now)` or `const [now,] = useState(Date.now())` for this. Does React.memo() render that moot or am I missing something?

Reply:
Yes, but isn't it good to use the current time? Then the graph's x axis is more accurate.

Reply:
Well, the `useMemo` on 14 means that this line won't run again unless the props change, so no, it won't get recalculated. But I agree with James -- that seems like a bug, not a feature. I would rather use the current time. Not enough to re-render the whole graph, though, which is quite slow.